src/App.tsx

```diff
@@ -1,40 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Tabs } from './components/Tabs';
 import './App.scss';
 
+export interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [activeTab, setActiveTab] = useState(tabs[0]);
+
+  const handleTabClick = (tab: TabType) => {
+    setActiveTab(tab);
+  };
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/99403198/3a489125-4b18-4a28-b392-cf00739353e3)

Reply:
Please provide a hint how to implement.